src/Layout/ConnectedLayout.tsx

```diff
@@ -13,8 +13,13 @@ import { initializeComputationMenu } from "./ComputationMenu";
 import { initializePresentation } from "./PresentationMenu";
 import { stateIsInvalid } from "../store/version";
 import { rawParametersToParametersConverter } from "../requested-data/requested-data-utilities";
+import { RootState } from "../reducers";
 
-const mapDispatchToProps = (dispatch: any): LayoutProps => ({
+const mapStateToProps = (state: RootState): Pick<LayoutProps, "dataLoaded"> => ({
+    dataLoaded: state.requestedDataState.dataLoaded,
+});
+
+const mapDispatchToProps = (dispatch: any): Pick<LayoutProps, "initializeState"> => ({
```

Comment:
Either myself or @Log234 could probably find out how to type this properly (`dispatch`)